Capstone/frontend-employee/src/Pages/EmployeeDashboard/Organization/EmpDisplayEmployee.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+// import "../../../style/DIsplay.css";
+import '../../EmployeeDashboard/Organization/EmpDisplayEmployee.css'
+import { Link } from "react-router-dom";
+
+const EmpDisplayEmployee = () => {
+  const [employees, setEmployees] = useState([]);
+  const [doj, setdoj] = useState("");
+  const [projects, setProjects] = useState([]);
+  const [manager, setManager] = useState("");
+
+  useEffect(() => {
+    getAllEmployees();
+  }, []);
+
+  useEffect(() => {
+    getAllProjects();
+  }, []);
+  const getAllProjects = async () => {
+    try {
+      const response = await axios.get(
+        "http://localhost:8080/project/getAllProjects"
```

Comment:
Use common functions for call actions for GET, POST, PUT methods. And don't use endpoints directly like this.